### ISSUE TITLE
Classification to regression

### ISSUE THREAD

**Issue by cgebe** [code-terminator/DilatedRNN#4]
I have trained the DRNN on a classification task which overfits after about 12k steps with a hidden size of 512 and batch size of 1024. As soon as i adjust it to a regression task (basically changing the loss function to mse, 1 output dim), it takes _very_ long to converge. Did someone make similiar experiences on regression tasks?

**Comment by code-terminator** [comment]
I have not tried on a regression task.  Maybe you want to first to identify whether the problem comes from the model or the data?  I mean if other models can easily overfit your regression data?  If yes, then maybe you want to adjust the learning rate of DRNN and see if the problem comes from there.  If other models cannot easily overfit the data, then maybe you might need to consider to further process the data or reformulate your task.  Thanks.

**Comment by cgebe** [comment]
Yes, i keep investigating, probably give transformer/multimodel a try - will report back. Thank you

**Comment by RidhwanAmin** [comment]
Hi @cgebe , have you made some progress using dilated rnn for regression. I'm trying migrating the code to tensorflow 2.0 but has not succeeded yet. Hope to hear from you
